src/pages/sign-up-success.tsx

```diff
@@ -1,9 +1,12 @@
+import React from 'react'
 import Footer from '../components/Footer'
 import SignUpSuccessForm from '../components/SignUpSuccessForm'
 import Navbar from '../components/Navbar'
+import Meta from '../components/Meta'
 
 const SignUpSuccess = () => (
   <div className="layout-authentication">
+    <Meta title="SignUp Success | Cobda" />
```

Comment:
Regarding the file name `sign-up-success`so it means “sign up”. Then, we should go with `Sign Up Success` or just `Signup Success`. What do you think?

Reply:
Do you have any refs, cause this one is same as `facebook.com`. 

![image](https://user-images.githubusercontent.com/32285706/107481326-94cfe200-6bb0-11eb-98b7-987fd4226f79.png)

Reply:
I agreed with your suggestion. We will be using `Signup Success` instead. It will be renamed in another PR since this PR is mainly for improving SEO.